services/kafkaService.go

```diff
@@ -114,8 +114,8 @@ func (kSvc *KafkaServiceImpl) SendMessage(topic, data string) error {
 
 	// Finally try to send the message.
 	partition, offset, err := callSend(kSvc, producerMessage)
-	log.Info("Sending message", log.Data{"topic": producerMessage.Topic, "partition": partition, "offset": offset})
-	log.Trace("Sending message", log.Data{"message source": deltaData})
+	log.InfoC(contextId, "Sending message", log.Data{"topic": producerMessage.Topic, "partition": partition, "offset": offset})
+	log.TraceC(contextId, "Sending message", log.Data{"message source": deltaData}) // TODO: Don't log entire data blob
```

Comment:
Please don't log the entire delta object.

Reply:
Removed logging of the whole delta object